src/applications/rated-disabilities/components/RatingList/Section.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import ListItem from './ListItem';
+
+export default function Section({ ratings }) {
```

Comment:
Maybe rename this to `List` and rename `RatingList` to `RatingLists` because it makes more sense